awx/ui_next/src/components/PaginatedDataList/ToolbarDeleteButton.jsx

```diff
@@ -14,11 +14,11 @@ import { t } from '@lingui/macro';
 import AlertModal from '../AlertModal';
 import { KebabifiedContext } from '../../contexts/Kebabified';
 
-const requireNameOrUsername = props => {
```

Comment:
EE does not have the `name` field. That is why I decided to rename this function.